### PR TITLE
feat: moderate project media

### PR DESCRIPTION
**Pull request by sk-yagi**

### CODE REVIEW

**Comment by sk-yagi** [comment]
Missing tag [resync] in the commit message, my bad.

**Comment by sk-shishi** [comment]
> Missing tag [resync] in the commit message, my bad.

Name it something like `feat: [resync] moderate project media`

**Comment by sk-shishi** [comment]
@sk-yagi Please rebase and remove the `[resync]` tag.

File: src/indexers/ai/project-moderation.ts [hunk 1]
```diff
@@ -225,29 +232,61 @@ export function aiProjectModerationIndexer(
...
+              "Content-Type": "application/x-www-form-urlencoded",
+            },
+            body: new URLSearchParams({
+              url: `${ipfsGatewayUrl}/ipfs/${cid}`,
+            }),
+          });
+          if (res.ok) {
+            const data = await res.json();
+            if (data == null)
+              throw new Error(`Response invalid: ${toJson(data)}`);
```
Comment (sk-shishi) [inline src/indexers/ai/project-moderation.ts:257]:
```ts
assert(data, "Data must not be empty");
```

What is "invalid"? 😛

**Comment by sk-shishi** [comment]
NOTE: Need to resync the `ai.project_moderation` table.

**Pull request merged**